testsuite/cpptests/test_parameter.h

```diff
@@ -35,7 +35,7 @@
 
 // Includes from nestkernel
 #include "nest_datums.h"
-#include "random.h"
+#include "../../nestkernel/random_generators.h"
```

Comment:
The nestkernel directory is already in the include path, so this can be
```suggestion
#include "random_generators.h"
```